docs/dashboard/visualization-date-and-time-controls.mdx

```diff
@@ -38,16 +42,49 @@ Click any of the following options in the **Quick Selector** to choose a commonl
 
 Click **Apply** to see metrics from your selected timeframe.
 
-## Choose a specific interval
+#### Choose a specific interval
 
 Beneath the Quick Selector is an input field and dropdown you use in combination to select a specific timeframe of
 minutes, hours, days, or months. Enter a number and choose the appropriate unit of time, then click **Apply**.
 
-## Choose multiple days
+#### Choose multiple days
 
 Use the calendar to select multiple days. Click on a date to begin the timeframe selection, then an ending date. The
 timeframe begins at noon on the beginning and end dates. Click **Apply** to see your selected multi-day timeframe.
 
+## Time controls
+
+The time control provides you the following options: **Play**, **Pause** and **Force Play**.
+* **Play** - the content of the page will be automatically refreshed while this is in the foreground
+* **Pause** - the content of the page isn't refreshed due to a manual request to pause it or, for example, when your investigating data on a 
+chart (cursor is on top of a chart)
+* **Force Play** - the content of the page will be automatically refreshed even if this is in the background
+
+With this, we aim to bring more clarity and allow you to distinguish if the content you are looking at is live or historical and also allow you
+ to always refresh the content of the page when the tabs are on the background. 
+
+Main use cases for **Force Play**:
+* You use a terminal or deployment tools to do changes in your infra and want to see immediately, Netdata is on the background, the impact 
+of these changes
+* You want to have Netdata on the background, example displayed on a TV, to constantly see metrics through dashboards or to watch the alert 
+status
+
+![The time control with Play, Pause and 
+Force Play](https://user-images.githubusercontent.com/82235632/129206460-03c47d0d-1a5b-428a-b972-473718b74bdb.png)
+
+## Timezone selector
+
+With the timezone select you have the ability to change the timezone on Netdata Cloud. More often than not are 
```

Comment:
```suggestion
With the timezone select, you have the ability to change the timezone on Netdata Cloud. More often than not are, there's a distribution in different timezones, requiring association. 
```

Reply:
here it was missing "teams", so it should read "More often than not team are distributed in different timezones and they need to collaborate."